src/packing.cc

```diff
@@ -1631,7 +1631,7 @@ size_t xnn_packed_stride_kai_qs4_weights_and_biases(
     size_t extra_bytes) {
   const uint32_t kr = UINT32_C(1) << gemm_config->log2_kr;
   const uint32_t sr = UINT32_C(1) << gemm_config->log2_sr;
-  return kai_get_rhs_packed_stride_rhs_pack_kxn_qsi4cxp_qsu4cxs1s0(k, /*nr=*/1,
+  return kai_get_rhs_packed_stride_rhs_pack_kxn_qsi4cxp_qs4cxs1s0(k, /*nr=*/1,
```

Comment:
Just a rename? No change in the zp==8 and packing 2 unsigned values?

Reply:
yea this seems like purely a renaming. They no longer assert zero_point==8, but we pass that in anyways. So I believe this is purely a renaming for us. Packing 2 unsigned values stays the same

https://gitlab.arm.com/kleidi/kleidiai/-/commit/49f0ca8b20505e9a5a2c1449a8f715b587593280

for reference

Reply:
Nice. Thanks for fixing this.